app/views/programs/show.html.haml

```diff
@@ -0,0 +1,45 @@
+%div
+  %h3
+    Program
+  %p
+    The largest official gathering of the year, RailsConf brings together top talent, companies, and project representatives from around the world. Learn and build with the best in sessions, workshops, keynotes and parties.
+  %ul.program_nav
+    %li.program-keynotes
+      Keynotes
+      %small
+        5
+    %li.program_sessions
+      %a{:href=>'#'}
+        Talks
+        %small
+          = @program_sessions.regular_sessions.count
+    %li.program-workshops
+      %a{:href=>'#'}
+        Workshops  Talks
+        %small
+          = @program_sessions.workshops.count
+
+#sessions
+  #regular_sessions
+    %nav#tracks-nav
+      %strong
+        Tracks
+      %ul
+        - current_event.tracks.each do |track|
+          %li
+            %a.track{ href: "#" }
+              = track.name
+        %li.back-to-top
+          %a{ href: "#" }
+            &uparrow; Back To Top
+    %h2
+      Talks
+    - @program_sessions.regular_sessions.each do |session|
+      %div
+        = render partial: 'session', locals: { session: session }
```

Comment:
@codyjamesbrooks it is probably best to use `program_session` instead of `session` here for a couple reasons. First of all there are certain names that can get you into trouble and `session` is one of them because it represents a lot in the context of a web request. I am guessing it is one of the reasons why CFPapp choose to clunky ProgramSession name for the model instead of just Session. 

Secondly if you also rename the partial to `_program_session.html.haml` it would give you the handy shorthand: 
```
render program_session
```

Reply:
Got it. Thanks. I will do the same thing over on the schedulePR as well.